documentation/content/Supported Plugins/galleriaio.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: Creating a Photo Gallery Article from Instagram post
+Tags: pelican-theme, pelican-plugin, photo gallery, instagram, galleriaio
+Category: Supported Plugins
+Date: 2020-01-27 17:40
+Slug: how-to-use-galleriaio-plugin
+Subtitle:
+Summary: Elegant can be configured to provide a simple display of pictures.
+Keywords: photos, gallery, photogallery, instagram, galleriaio
+Authors: Pablo Iranzo Gómez
+instagram: B3ePmL7gd1P
+---
+
+[TOC]
+
+Similar to the use case defined in [Photogallery]({filename}photogallery.md), Pelican uses <https://galleria.io> code to show images from instagram.
+
+## Article Metadata
+
+To enable the plugin, just set a `yaml preamble` entry for instagram like:
+
+```yaml
+instagram: <picture ID>
```

Comment:
Is it a picture ID or a gallery ID?

Reply:
As described in the doc, it's the part in the url, either a single picture or a set of (max 10 pictures I think) that gets shown, the javascript handles both and prefills the list of pictures

Reply:
Can you please add this information to the doc, that link can be either to a single image or a bunch of images.